addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/envisalinkbridge.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dscalarm"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
+
+	<bridge-type id="envisalinkbridge">
+		<label>EyezOn Envisalink Bridge</label>
+		<description>This bridge represents the EyezOn Envlisalink 3/2D Ethernet interface.</description>
+
+        <channels>
+            <channel id="bridge_connection" typeId="bridge_connection"/>
+        </channels>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network_address</context>
+				<label>Envisalink Bridge IP</label>
```

Comment:
we are already in the context of the Envisalink, so just call it "IP address" or "Hostname".